impeller/entity/contents/solid_color_contents.cc

```diff
@@ -25,20 +23,19 @@ const Color& SolidColorContents::GetColor() const {
   return color_;
 }
 
-void SolidColorContents::SetPath(Path path) {
-  path_ = std::move(path);
-}
-
-void SolidColorContents::SetCover(bool cover) {
-  cover_ = cover;
+void SolidColorContents::SetGeometry(std::unique_ptr<Geometry> geometry) {
+  geometry_ = std::move(geometry);
 }
 
 std::optional<Rect> SolidColorContents::GetCoverage(
     const Entity& entity) const {
   if (color_.IsTransparent()) {
     return std::nullopt;
   }
-  return path_.GetTransformedBoundingBox(entity.GetTransformation());
+  if (geometry_ == nullptr) {
```

Comment:
This worked before because we'd end up with an empty path here if nothing was set, and we have a test that verifies this works for SolidColorContents, but nothing else.

Is there a valid use-case for drawing a contents without a path or cover, or should this assert instead?